astropy/utils/console.py

```diff
@@ -887,7 +887,7 @@ def __init__(self, msg, color='default', file=None, step=1,
             completely silent.
 
         step : int, optional
-            Only update the spinner every *step* steps
+            Only update the spinner every *step* steps. Must be >= 0.
```

Comment:
Please make this explicit by raising a `ValueError` if `step < 1`.

Reply:
Done